components/expenses/Expense.js

```diff
@@ -185,6 +188,13 @@ class Expense extends React.Component {
     this.setState({ modified: false, mode: 'details' });
   }
 
+  handleErrorMessage(errorMessage) {
+    console.error(errorMessage);
```

Comment:
Looks like we're logging the error two times

Reply:
You're right, shouldn't have logged that there.